.kokoro/build-docs.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
You can remove this file and the config `docs.cfg`. The publish to googleapis.dev is done by `publish-docs.cfg` and `publish-docs.sh`